spec/controllers/locations_controller_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe LocationsController do
+  render_views
+
+  describe '#index' do
+    let!(:location) { FactoryGirl.create(:location, uid: '25de9301-50b5-49ba-a5da-7f40a2fcfe29') }
```

Comment:
Given the need to eagerly load this, plus the fact it isn't actually referenced by name in this spec, perhaps it would be clearer if this setup was in a `before` block instead?